parts/windows/windowscsehelper.ps1

```diff
@@ -275,7 +275,21 @@ function Get-WindowsVersion {
     switch ($buildNumber) {
         "17763" { return "1809" }
         "20348" { return "ltsc2022" }
-        "25398" { return "ltsc2022" } # Only for test of partner team now. Will remove it further.
+        "25398" { return "ltsc2022" }

```

Comment:
I think that we should not return ltsc2022 for 23H2 since we have separated the code for pause image